internal/server/handlers/render_test.go

```diff
@@ -71,3 +72,35 @@ func TestGetRenderHandler(t *testing.T) {
 	assert.True(t, strings.Contains(body, "CounterMetric2"))
 	assert.True(t, strings.Contains(body, "200"))
 }
+
+func ExampleMetricHandler_Render() {
```

Comment:
It should be in the `example` package.
https://github.com/golang-standards/project-layout/tree/master/examples

Reply:
moved